src/__tests__/ThemeHandler.test.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import { shallow } from "enzyme";
+import ThemeHandler from "../components/ThemeHandler";
+
+describe("<themehandler /> rendering", () => {
+    const wrapper = shallow(<ThemeHandler />);
+    // test if the TopNav component renders
+    it("should be able to render <themehandler />", () => {
+        wrapper;
```

Comment:
Does this actually test the wrapper? Shouldn't you use an expect() + matcher to test?